mapreduce/tests/compose/docker-compose.yaml

```diff
@@ -0,0 +1,13 @@
+version: "3"
+
+services:
+  hadoop-single-node:
+    image: sequenceiq/hadoop-docker:2.7.1
+    hostname: ${HOSTNAME}
+    container_name: dd-test-mapreduce
+    environment:
+      - CLUSTER_NAME=MapReduceCluster
+    ports:
+      - "8088:8088"
+    volumes:
+      - './yarn-site.xml:/usr/local/hadoop/etc/hadoop/yarn-site.xml'
```

Comment:
Minor, but seems ending newlines are missing for some files. You may want to configure your IDE to add it automatically.

Same for:
- mapreduce/tox.ini
- mapreduce/tests/compose/yarn-site.xml